src/screens/ModalMoreOptions.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import {
+  Text,
+  View,
+  TouchableWithoutFeedback,
+  SafeAreaView,
+  FlatList,
+  ScrollView,
+  StyleSheet,
+  Image
+} from 'react-native';
+
+import PropTypes from 'prop-types';
+import { device, gStyle, images, colors, fonts } from '../constants';
+import moreOptions from '../mockdata/menuMoreOptions';
+import LineItemCategory from '../components/LineItemCategory';
+
+export default function ModalMoreOptions({
+  navigation,
+  screenProps: { setToggleTabBar }
+}) {
```

Comment:
this can be this:
```const ModalMoreOptions = ({ navigation, screenProps: { setToggleTabBar } }) => {```
and then at the end of this file add:
`export default ModalMoreOptions;`